i18n/json/src/components/messages.json

```diff
@@ -0,0 +1,62 @@
+[
```

Comment:
Do we need these files in both places? I only see them once in elements

Reply:
This is mostly to separate out the duplicated messages and components